niworkflows/utils/spaces.py

```diff
@@ -0,0 +1,114 @@
+"""Utilities for tracking and filtering spaces"""
+
+FSAVERAGE_DENSITY = {
+    '642': 'fsaverage3',
+    '2562': 'fsaverage4',
+    '10k': 'fsaverage5',
+    '41k': 'fsaverage6',
+    '164k': 'fsaverage',
+}
+NONSTANDARD_REFERENCES = ['anat', 'T1w', 'run', 'func', 'sbref', 'session', 'fsnative']
```

Comment:
Do we want these two to be under the `Spaces` namespace as class attributes?

Reply:
I wanted to make the object as lightweight as possible - do you think we should include these as part of the class?

Reply:
I guess, because these names are not standard (there's only a concept of individual space in BIDS), a class attribute is cleaner to update by applications